packages/client/package.json

```diff
@@ -47,14 +44,13 @@
     "@types/react-dom": "^18.2.17",
     "@types/react-syntax-highlighter": "^15.5.11",
     "@types/turf": "^3.5.32",
-    "babel-jest": "^29.7.0",
     "d3-scale": "^4.0.2",
     "jest": "^29.7.0",
```

Comment:
@dwnoble This file and static/package.json has some common deps. I wonder if some more can be removed from here?